test/integration/targets/template/tasks/main.yml

```diff
@@ -120,6 +120,74 @@
         - 'import_as_with_context_diff_result.stdout == ""'
         - "import_as_with_context_diff_result.rc == 0"
 
+# VERIFY trim_blocks
+
+- name: Render a template with "trim_blocks" set to False
+  template: src=trim_blocks.j2 dest={{output_dir}}/trim_blocks_false.templated trim_blocks=False
```

Comment:
Please use multi-line YAML rather than `k=v` syntax. I know the rest of the tasks use `k=v` syntax, but they need to be changed at some point.

Reply:
Noted, it's just that my commit for `trim_blocks` fixes exactly that, that the `k=v` syntax for `trim_blocks` was not working. I will gladly change to multi-line syntax though if you insist.